frontend/apps/app/components/Chat/services/aiMessageService.ts

```diff
@@ -157,11 +167,8 @@ export const createAndStreamAIMessage = async ({
     )
 
     // Parse JSON response with type safety
-    const data = (await response.json()) as {
-      success?: boolean
-      text?: string
-      error?: string
-    }
+    const rawData = await response.json()
+    const data = parse(ChatAPIResponseSchema, rawData)
```

Comment:
If parse is used, an exception is thrown, so error handling should be considered, but since it seems to be thrown in other places, there should be no problem.